plugins/modules/dcnm_vrf.py

```diff
@@ -293,6 +356,9 @@ def __init__(self, module):
         self.fabric = module.params['fabric']
         self.config = copy.deepcopy(module.params.get('config'))
         self.check_mode = False
```

Comment:
Not really in scope for this PR but do we have tests for check_mode?

Reply:
i havent done it before, will discuss with you on the call.

Reply:
Added the check mode cases in ut/it and verified the results.